src/coreclr/jit/importer.cpp

```diff
@@ -1779,6 +1780,13 @@ GenTree* Compiler::impNormStructVal(GenTree*             structVal,
             alreadyNormalized = true;
             break;
 
+        case GT_CNS_VEC:
+        {
+            assert(varTypeIsSIMD(structVal) && (structVal->gtType == structType));
+            makeTemp = true;
```

Comment:
Given we `makeTemp = true;` here, I wonder if the rationalization code that strips `OBJ(ADDR(SIMD))` to just `SIMD` is needed for `CNS_VEC`.

Reply:
IIRC, this was needed because of handles again.